docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -474,7 +474,7 @@ relation to port exhaustion.
 - `ALLOW_ONLY_EXTERNAL_REGISTRATION`: **false** Set to true to force registration only using third-party services.
 - `NO_REPLY_ADDRESS`: **DOMAIN** Default value for the domain part of the user's email address in the git log if he has set KeepEmailPrivate to true.
   The user's email will be replaced with a concatenation of the user name in lower case, "@" and NO_REPLY_ADDRESS.
-- `USER_DELETE_WITH_COMMENTS_MAX_DAYS`: **0** If a user deletes his account before that amount of days, his comments will be deleted as well.
+- `USER_DELETE_WITH_COMMENTS_MAX_TIME`: **2h** If a user deletes his account before that amount of days, his comments will be deleted as well. 0 to disable it.
```

Comment:
Really unsure if this is a good idea at all

Reply:
@CirnoT do you think it should be disabled by default?

Reply:
I do believe this solution is a very specific use-case one and indeed should not be enabled by default.

Proper interaction limits like GH has would be a better solution and would make this one unnecessary in future.

Reply:
I agree this should be disabled by default